openapi.yml

```diff
@@ -10,6 +10,32 @@ servers:
     description: TIS Self-Service
 
 paths:
+  /trainee/cct/calculate:
+    post:
+      summary: Return the CCT Calculation with the CCT date calculated.
+      operationId: getCctCalculated
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/cctCalculationDetail"
+        required: true
+      responses:
+        200:
+          description: Ok
+          headers:
+            location:
+              description: The location of the returned calculation
+              schema:
+                type: string
+                format: uri
+                example: /api/trainee/cct/calculate
```

Comment:
The headers/location should be removed, since we don't persist anything that can be retrieved.

Reply:
True, sorry, missed that.